CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,36 @@
+#Code of Conduct
+##Our Pledge
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+##Our Standards
+Examples of behavior that contributes to creating a positive environment include:
+<ul>
```

Comment:
Oh, you can make a list in markdown with * (as bullet points):

This would look like this:

* Being respectful of differing viewpoints and experiences
* Gracefully accepting constructive criticism
* Focusing on what is best for the community
* Showing empathy towards other community members

But in markdown is actually:

```
* Being respectful of differing viewpoints and experiences
* Gracefully accepting constructive criticism
* Focusing on what is best for the community
* Showing empathy towards other community members
```